### ISSUE TITLE
Markos Charatzas - 1 product, 3 years, one shipment. A true story.

### ISSUE THREAD

**Issue by qnoid** [techmeetup/edinburgh#52]
This is a talk that gave at http://thehighlandfling.com last year.
## Summary 

This is a true story about the life lessons learned over 3 years it took to ship 1 product.
Will provoke you, ask you to make up your mind and be honest about what you want as you face the ugly truth.
Will explain in plain english why you have to release and what you risk if you don’t while showing you were to focus your energy.
A controversial, passionate talk that will make you come out a better human.

**Comment by mikemcquaid** [comment]
@qnoid confirmed on Twitter he'll speak in March 2015.

**Comment by qnoid** [comment]
ditto

**Comment by mikemcquaid** [comment]
@qnoid will you need internet/a laptop for your talk? what connector does your laptop have for the project? Thanks!

**Comment by qnoid** [comment]
Have a laptop. Don't need Internet. 
Have HDMI port and a thunderbolt one.
Don't have any adaptors or cables.

**Comment by mikemcquaid** [comment]
@qnoid Perfect, thanks, we have everything you'd need.

**Comment by MariusCiocanel** [comment]
:ship: d!
